state/migration_export_test.go

```diff
@@ -1146,9 +1147,7 @@ func (s *MigrationExportSuite) newResource(c *gc.C, appName, name string, revisi
 func (s *MigrationExportSuite) TestRemoteApplications(c *gc.C) {
 	// NOTE: the key 'c#<name>' isn't overly useful for someone looking
```

Comment:
It was a comment that Tim added about the global key for remote applications - I guess he found leftover rows in the status collection and was trying to work out what collection they belonged to. Should I change the global key prefix for remote applications to something a bit easier to search for?